lib/translate/common.cc

```diff
@@ -31,8 +32,122 @@ void DebugLog(const char* format, ...) {
   va_end(args);
 }
 
-void SetDebugCallback(void (*callback)(const char*)) {
+void SetDebugCallback(void (*callback)(const char *)) {
   debug_callback = callback;
 }
 
+StatusCode MakeScsiOpcode(uint8_t val, scsi_defs::OpCode &opcode) {
+  if (val > 0xaf) {
+    char debug_buffer[100];
+    sprintf(debug_buffer, "invalid opcode. %ux is out of range.", val);
+    if (debug_callback != NULL) debug_callback(debug_buffer);
+    return StatusCode::kInvalidInput;
+  }
+  opcode = static_cast<scsi_defs::OpCode>(val);
+  return StatusCode::kSuccess;
+}
+
+const char *ScsiOpcodeToString(scsi_defs::OpCode opcode) {
+  uint32_t opcode_val = static_cast<uint32_t>(opcode);
+  switch (opcode_val) {
+    case 0x0:
+      return "kTestUnitReady";
+      break;
```

Comment:
we don't need the breaks since there's the return. I'm also not sure what this is used for, to have more clear logs?